generated/nirfmxlte/nirfmxlte.proto

```diff
@@ -2582,8 +2582,8 @@ message CfgPSSCHModulationTypeRequest {
   nidevice_grpc.Session instrument = 1;
   string selector_string = 2;
   oneof modulation_type_enum {
-    PsschModulationType modulation_type = 3;
-    int32 modulation_type_raw = 4;
+    int32 modulation_type_raw = 3;
```

Comment:
We want to keep the modulation_type_raw field number as `3` so old clients will still work passing forward their ints for field `3` and it won't be interpreted as an enum value.